ES/ESmy020.xml

```diff
@@ -358,57 +330,66 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                           <note></note>
                            <list>
                               <item xml:id="q1" n="1">
-                                 <locus from="1r" to="5v"/>
+                                 <locus from="1r" to="4v"/>
                                  <dim unit="leaf">4</dim>
-                                  <locus from="1r" to="14v"/>
                                   s.l.: 1, no stub
                                   s.l.: 2, no stub
                                   s.l.: 3, no stub
                                   s.l.: 4, no stub
                                    </item>
+
                               <item xml:id="q2" n="2">
-                                 <locus from="6r" to="7v"/>
+                                 <locus from="5r" to="6v"/>
                                  <dim unit="leaf">2</dim>
-                                  <dim unit="leaf">2</dim>
-                                </item>
-                              <item xml:id="q3" n="3">
-                                 <locus from="6r" to="7v"/>
-                                 <dim unit="leaf">2</dim>>
-                                 <locus from="15r" to="22v"/>
-                                 <dim unit="leaf">8</dim>
                                </item>
-                              <item xml:id="q4" n="4">
+
+                               <item xml:id="q3" n="3">
+                                  <locus from="7r" to="14v"/>
+                                   <dim unit="leaf">8</dim>
+                                  </item>
+
+                                  <item xml:id="q4" n="4">
+                                     <locus from="15r" to="22v"/>
+                                      <dim unit="leaf">8</dim>
+                                     </item>
+
+                              <item xml:id="q5" n="5">
                                  <locus from="23r" to="33v"/>
-                                  <dim unit="leaf">11</dim>
-                                  s.l.: 11, stub after 10
+                                  <dim unit="leaf">12</dim>
+                                  1 missing, no stub
```

Comment:
this syntax does not correspond to the rules
we do not describe here missing leaves, only what is there
do you mean s.l.: 12, no stub?
you can add a note explaining that 33 is single because its counterpart was lost